Havit.Blazor.Components.Web.Bootstrap/Navigation/HxSidebarItem.razor.cs

```diff
@@ -48,10 +48,26 @@ public partial class HxSidebarItem : IAsyncDisposable
 		/// </summary>
 		[Parameter] public string CssClass { get; set; }
 
+		/// <summary>
+		/// Inner custom content for the <see cref="HxSidebarItem"/>.
+		/// </summary>
+		[Parameter] public RenderFragment ContentTemplate { get; set; }
+
 		/// <summary>
 		/// Sub-items (not intended to be used for any other purpose).
 		/// </summary>
-		[Parameter] public RenderFragment ChildContent { get; set; }
+		[Parameter] public RenderFragment SubItems { get; set; }
+
+		/// <summary>
+		/// Use the <see cref="SubItems"/> parameter instead as <see cref="ChildContent"/> is deprecated.
+		/// </summary>
+		[Parameter]
+		[Obsolete("ChildContent is obsolete, please use the SubItems parameter instead.")]
+		public RenderFragment ChildContent
+		{
+			set => SubItems = value;
+			get => SubItems;
+		}
```

Comment:
Keep the parameter as `ChildContent` without the move to `SubItems`. I found this pattern in other libraries and makes sense for me. We might consolidate our other components later.